crates/bevy_render/src/mesh/shape/cone.rs

```diff
@@ -0,0 +1,137 @@
+use crate::mesh::{Indices, Mesh};
+use wgpu::PrimitiveTopology;
+
+/// A cone which stands on the XZ plane
+#[derive(Clone, Copy, Debug)]
+pub struct Cone {
+    /// Radius in the XZ plane at the upper face.
+    pub top_radius: f32,
+
+    /// Radius in the XZ plane at the bottom face.
+    pub bottom_radius: f32,
+
+    /// Height of the cone in the Y axis.
+    pub height: f32,
+    /// The number of vertices around each horizontal slice of the cone. If you are looking at the cone from
+    /// above, this is the number of points you will see on the circle.
+    /// A higher number will make it appear more circular.
+    pub resolution: u32,
+    /// The number of segments between the two ends. Setting this to 1 will have triangles spanning the full
+    /// height of the cone. Setting it to 2 will have two sets of triangles with a horizontal slice in the middle of
+    /// the cone. Greater numbers increase triangles/slices in the same way.
+    pub segments: u32,
+}
+
+impl Default for Cone {
+    fn default() -> Self {
+        Self {
+            top_radius: 0.2,
```

Comment:
These are really weird default settings. Surely a top radius of 0.0 and a bottom radius of 1.0 is more sensible?

Reply:
Changed to your suggestion.
I think in my mind I was fixated on showing both top/bottom faces.